.github/workflows/node.js.yml

```diff
@@ -0,0 +1,36 @@
+# This workflow will do a clean install of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: test coverage
+
+on:
+  pull_request:
+    branches: 
+      - develop
+      - main
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x, 16.x]
```

Comment:
The challenge here is that the GH workflow does not support node version 17.x, which Factiverse design system supports. Thoughts?

Reply:
I've added 17.x as well.